code/modules/modular_computers/file_system/programs/paperworkprinter.dm

```diff
@@ -0,0 +1,75 @@
+//TODO: add more sane printing handling
+/datum/computer_file/program/paperwork_printer
+	filename = "ppwrkprnt"
+	filedesc = "Paperwork Printing"
+	category = PROGRAM_CATEGORY_MISC
+	program_icon_state = "id"
+	extended_desc = "Program for dispensing paperwork"
+	requires_ntnet = FALSE
+	size = 4
+	tgui_id = "NtosPaperworkPrinter"
+	program_icon = "clipboard-list"
+	var/obj/item/paper/paperwork/which
+	var/print = 1
```

Comment:
- why is this declared here instead of the sole scope that it's used?
- why does it start out TRUE?
- why does it use 1 instead of TRUE when it's used solely in truthy operations?

Reply:
same for the `which` variable, it has no reason to be declared here

Reply:
> It uses 1 instead of TRUE because it is functionally equivalent according to byond documentation, and is more readable in my opinion. 

you are correct in that they are functionally equivalent; TRUE is a macro, which means it does nothing but point to `1`; it was created solely for readability and ease of use.

> It's a bit flag, if that makes sense.

it doesn't because the only way you use it is in truthy operations `if(print)` and `print = !print`

further, BYOND fully supports bitflags e.g. `1<<0` `1<<1` `1<<2` and bitwise operations, see [byond ref](https://www.byond.com/docs/ref/) for further info, although i don't immediately see why you'd use them here